dev-images/vagrant/Vagrantfile

```diff
@@ -16,15 +17,18 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         libvirt.cpus = 1
         libvirt.cpu_mode = 'host-passthrough'
         libvirt.nested = true
-        
+
         libvirt.management_network_name = 'leapp-devenv-network'
         libvirt.management_network_address = '192.168.124.0/24'
     end
 
-    config.vm.provision "shell", inline: "dnf -y update && dnf -y install docker libvirt"
+    config.vm.provision "shell", inline: "dnf -y upgrade && dnf -y install docker"
 
-    config.vm.provision "shell", inline: "cd #{HOME} && curl -O #{BUILD_SCRIPT_URL} && $SHELL #{BUILD_SCRIPT} 2>&1 | tee #{HOME}/leapp-setup.log", privileged: false
-    config.vm.provision "shell", inline: "systemctl enable docker libvirtd"
-    config.vm.provision "shell", inline: "systemctl start docker libvirtd"
+    config.vm.provision "shell", inline: "dnf install -y git ansible && "\
```

Comment:
yum shouldn't be used here, since this should be run with parameter: privileged: false. I suggest to move the dnf commands to the line above

Reply:
@GAZDOWN what privileged false?

Reply:
we do not want to run the complete ansible playbook with root privileges during the provision phase. In other words, we want to have everything available for vagrant user. When it comes to installation of binaries and dependencies, where you need root privileges, ansible will take care of this (become).

Reply:
I was trying to be consistent with the Dockerfile, so the upgrade and docker installation is separated from the rest (git & ansible).